apps/andi/lib/andi_web/live/edit_live_view/data_dictionary_form.ex

```diff
@@ -217,12 +221,8 @@ defmodule AndiWeb.EditLiveView.DataDictionaryForm do
   end
 
   def handle_event("overwrite-schema", _, %{assigns: %{pending_changeset: nil}} = socket) do
-    changeset_with_error =
-      socket.assigns.changeset
-      |> Map.put(:action, :update)
-      |> Changeset.add_error(:schema_sample, "There was a problem interpreting this file")
-
-    {:noreply, assign(socket, changeset: changeset_with_error)}
+    socket.assigns.changeset
+    |> send_error_interpreting_file(socket)
```

Comment:
This did not originally set `loading_schema` to false. Not sure if that makes a difference